src/org/omegat/util/Preferences.java

```diff
@@ -234,6 +234,10 @@
      * Match to insert
      */
     public static final int BEST_MATCH_MINIMAL_SIMILARITY_DEFAULT = 80;
+
+    public static final String PENALTY_FOR_NON_TARGET = "penalty_non_target";
```

Comment:
I think this is a bit cryptic when viewed without knowing the context of how it was added. Could you add some explanation as a comment?

Reply:
Done.